vnext/Shared/Shared.vcxitems

```diff
@@ -579,4 +580,4 @@
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
     </ClCompile>
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
I really wish Visual Studio added a trailing newline at the end of the file, so the rest of us that hand edit these files don't have to keep dealing with linters adding these spurious newlines 😅